File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.js
function solution(arr1, arr2) {
    var answer = 0;
    if(arr1.length > arr2.length) answer = 1;
    else if(arr1.length < arr2.length) answer = -1;
    else{
        const sum1 = arr1.reduce((acc, cur, idx) => acc += cur);
        const sum2 = arr2.reduce((acc, cur, idx) => acc += cur);
        if(sum1 > sum2) answer = 1;
        else if(sum1 < sum2) answer = -1;
        else answer = 0;
    }
    return answer;
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.py
def solution(arr1, arr2):
    len1 = len(arr1)
    len2 = len(arr2)
    sum1 = sum(arr1)
    sum2 = sum(arr2)
    if len1 == len2: 
        if sum1 == sum2: return 0
        if sum1 > sum2: return 1
        return -1
    if len1 > len2: return 1
    return -1